recipes-bsp/u-boot/u-boot-fslc_2020.10.bb

```diff
@@ -6,7 +6,9 @@ order to provide support for some backported features and fixes, or because it \
 was submitted for revision and it takes some time to become part of a stable \
 version, or because it is not applicable for upstreaming."
 
-DEPENDS_append = " bc-native dtc-native lzop-native"
+inherit ${@oe.utils.ifelse(d.getVar('UBOOT_PROVIDES_BOOT_CONTAINER') == '1', 'imx-boot-container', '')}
```

Comment:
Couldn't we drop the new UBOOT_PROVIDES_BOOT_CONTAINER. This location seems to be the only user of it and it could directly test for the override.

inherit ${@bb.utils.contains('OVERRIDES', 'boot-container', 'imx-boot-container', '')}

Reply:
Bad idea as OVERRIDES use : separator.

Reply:
Correct, this I've already tried out and realized _OVERRIDES_ would not be usable here. It would require `.split()` function and then the construct would become quite unusable.